network/p2p/throttler_handler.go

```diff
@@ -46,9 +42,9 @@ func (t ThrottlerHandler) AppGossip(ctx context.Context, nodeID ids.NodeID, goss
 	t.handler.AppGossip(ctx, nodeID, gossipBytes)
 }
 
-func (t ThrottlerHandler) AppRequest(ctx context.Context, nodeID ids.NodeID, deadline time.Time, requestBytes []byte) ([]byte, error) {
+func (t ThrottlerHandler) AppRequest(ctx context.Context, nodeID ids.NodeID, deadline time.Time, requestBytes []byte) ([]byte, *common.AppError) {
 	if !t.throttler.Handle(nodeID) {
-		return nil, fmt.Errorf("dropping message from %s: %w", nodeID, ErrThrottled)
+		return nil, ErrThrottled
```

Comment:
the nodeID would now get lost; I can't really say if that's a valuable information or not.

Reply:
We can add context to this error